src/core/include/openvino/op/squeeze.hpp

```diff
@@ -31,9 +35,11 @@ class OPENVINO_API Squeeze : public Op {
     std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& new_args) const override;
 
     bool is_dynamic() const override;
+    bool get_pytorch_dynamic_rank() const;
```

Comment:
Update:
docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/shape/squeeze-1.rst

Attribute name specified here should be used for getter/setter to use same name.

Reply:
done